<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaCadastroCliente</class>
 <widget class="QDialog" name="telaCadastroCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BIBLIOTECA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/images/images/logo.ico</normaloff>:/images/images/logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 13px;
border-radius: 5px;
background-color:#49b1f7;
color: #fefefe;
font-family:Franklin Gothic Heavy;
}

:hover{
background-color:#73c3f9;
}</string>
   </property>
   <property name="text">
    <string>&lt; Voltar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>371</width>
     <height>322</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>NOME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nomeCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>IDADE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idadeCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>TELEFONE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="telefoneCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>E-MAIL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>ENDEREÇO (RUA E NÚMERO)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="enderecoCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
        </property>
        <property name="text">
         <string>CPF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="documentoCliente_txt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="cliente_img">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="arquivos.qrc">:/images/images/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cadastrarCliente_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 20px;
border-radius: 5px;
background-color:#49b1f7;
color: #fefefe;
font-family:Franklin Gothic Heavy;}

:hover{
background-color:#73c3f9;
}
</string>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#49b1f7;
color:#fefefe;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="login_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:Franklin Gothic Heavy;
font-size:25px;
color:#fefefe;
</string>
    </property>
    <property name="text">
     <string>CADASTRAR CLIENTE</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>cliente_img</zorder>
  <zorder>cadastrarCliente_btn</zorder>
 </widget>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
